packages/web-app-files/src/components/FilesList/KeyboardActions.vue

```diff
@@ -40,13 +41,7 @@ export default defineComponent({
   },
 
   mounted() {
-    for (const elementId of this.keybindOnElementIds) {
-      const element = document.getElementById(elementId)
-      if (element) {
-        element.addEventListener('keydown', this.handleSelectionShortcuts, false)
-      }
-    }
-    document.addEventListener('keydown', this.handleClipboardShortcuts)
+    document.addEventListener('keydown', this.handleShortcuts)
```

Comment:
Since you made all event handlers global now, can you delete the `keybindOnElementIds` prop now?